boards/st/stm32l562e_dk/doc/index.rst

```diff
@@ -160,6 +160,9 @@ hardware features:
 +-----------+------------+-------------------------------------+
 | DAC       | on-chip    | DAC Controller                      |
 +-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | TFT LCD screen with st7789v driver  |
```

Comment:
The FMC peripheral which  controls the TFT -LCD driver is  **on-chip**. Does it actually make the display to be "on-chip" or "on-board"